### ISSUE TITLE
pgwire: DInterval conversion missing on binary protocol in 2.0

### ISSUE THREAD

**Issue by knz** [cockroachdb/cockroach#28134]
Reported by @mostov on gitter

- Which version of CockroachDB are you using?

cockroach-v2.0.4.windows-6.2-amd64

- What did you do?

```sql
CREATE TABLE tbl_time_gift (
user_id STRING(128) NOT NULL,
next_gift_time TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
FAMILY "primary" (user_id, next_gift_time, rowid)
);
-- ...
select age(next_gift_time ) from tbl_time_gift;
```

- What did you expect to see?

successful result

- What did you see instead?

cockroachdb crashes

**Comment by knz** [comment]
User reports that even `select age(now())` crashes, no table is needed

**Comment by knz** [comment]
So I found that 2.0.4 is missing the DInterval pgwire code which 2.1 has. 

This was PR #25242 but apparently was not backported to 2.0. Will do.

**Comment by BramGruneir** [comment]
I just tested this on Windows 
Edition: Windows 10 pro
Version: 1803
OS Build: 17134.165

And I did not get any crashes for both the individual line or using the table.

```sql

root@:26257/test> select age(now());
+-----+
| age |
+-----+
| 0s  |
+-----+
(1 row)

Time: 3.147ms

root@:26257/test> CREATE TABLE tbl_time_gift (
               -> user_id STRING(128) NOT NULL,
               -> next_gift_time TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
               -> FAMILY "primary" (user_id, next_gift_time, rowid)
               -> );
CREATE TABLE

Time: 159.2746ms

root@:26257/test> select age(next_gift_time ) from tbl_time_gift;
+-----+
| age |
+-----+
+-----+
(0 rows)

Time: 58.0112ms
```

**Comment by knz** [comment]
@BramGruneir thanks the problem occurs with the binary protocol

**Comment by BramGruneir** [comment]
Yep, I noticed that after I had already run the tests.  Figured I'd post the results anyway.

**Comment by BramGruneir** [comment]
We should consider adding "What did you use to connect to the database (if applicable)?" to our template.

**Comment by knz** [comment]
Fixed in #28135.
